module/zfs/abd.c

```diff
@@ -696,6 +697,7 @@ abd_return_buf_copy(abd_t *abd, void *buf, size_t n)
 {
 	if (!abd_is_linear(abd)) {
 		abd_copy_from_buf(abd, buf, n);
+		abd_update_borrow_stats(abd);
```

Comment:
I'm sort of indifferent at this point. I thought it would be more useful than it was, however it was useful to spot a bug on ARM, but only because it was a couple of orders of magnitude different to what it should have been.

I do want to do a more comprehensive stats package for `vdev_disk` proper, and this state probably is better done there.

@behlendorf you suggested this earlier. Would you rather keep it or remove it? I am fairly ambivalent.

Reply:
I wouldn't object to simply dropping this.  I also initially thought it would be more useful.

Reply:
And there it goes... :balloon: